medusa/reconstruct/expand.py

```diff
@@ -1,261 +1,72 @@
+    return reaction_probability
 
 def gapfill_to_ensemble(model, iterations=1, universal=None, lower_bound=0.05,
-                 penalties=None, exchange_reactions=False,
+                 penalties= None, exchange_reactions=False,
                  demand_reactions=False, integer_threshold=1e-6):
-    """
-    Performs gapfilling on model, pulling reactions from universal.
-    Any existing constraints on base_model are maintained during gapfilling, so
-    these should be set before calling gapfill_to_ensemble (e.g. secretion of
-    metabolites, choice of objective function etc.).
-
-    Currently, only iterative solutions are supported with accumulating
-    penalties (i.e. after each iteration, the penalty for each reaction
-    doubles).
-
-    Parameters
-    ----------
-    model : cobra.Model
-        The model to perform gap filling on.
-    universal : cobra.Model
-        A universal model with reactions that can be used to complete the
-        model.
-    lower_bound : float, 0.05
-        The minimally accepted flux for the objective in the filled model.
-    penalties : dict, None
-        A dictionary with keys being 'universal' (all reactions included in
-        the universal model), 'exchange' and 'demand' (all additionally
-        added exchange and demand reactions) for the three reaction types.
-        Can also have reaction identifiers for reaction specific costs.
-        Defaults are 1, 100 and 1 respectively.
-    integer_threshold : float, 1e-6
-        The threshold at which a value is considered non-zero (aka
-        integrality threshold). If gapfilled models fail to validate,
-        you may want to lower this value. However, picking a threshold that is
-        too low may also result in reactions being added that are not essential
-        to meet the imposed constraints.
-    exchange_reactions : bool, False
-        Consider adding exchange (uptake) reactions for all metabolites
-        in the model.
-    demand_reactions : bool, False
-        Consider adding demand reactions for all metabolites.
-
-    Returns
-    -------
-    ensemble : medusa.core.Ensemble
-        The ensemble object created from the gapfill solutions.
-    """
     gapfiller = GapFiller(model, universal=universal,
-                          lower_bound=lower_bound, penalties=penalties,
+                          lower_bound=lower_bound, penalties= 1-reaction_probability,
                           demand_reactions=demand_reactions,
                           exchange_reactions=exchange_reactions,
                           integer_threshold=integer_threshold)
     solutions = gapfiller.fill(iterations=iterations)
     print("finished gap-filling. Constructing ensemble...")
     ensemble = _build_ensemble_from_gapfill_solutions(model,solutions,
                                                     universal=universal)
-
     return ensemble
 
-def iterative_gapfill_from_binary_phenotypes(model,universal,phenotype_dict,
```

Comment:
We still need the iterative_gapfill_from_binary_phenotypes function--please add this back in your pull request